rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h

```diff
@@ -119,6 +119,17 @@ rcl_variant_t * rcl_yaml_node_struct_get(
   const char * param_name,
   rcl_params_t * params_st);
 
+/// \brief Get the descriptor struct for a given parameter
+/// \param[in] node_name is the name of the node to which the parameter belongs
+/// \param[in] param_name is the name of the parameter whose value is to be retrieved
```

Comment:
@bpwilcox this needs an update: 
```suggestion
/// \param[in] param_name is the name of the parameter whose descriptor is to be retrieved
```